recipes/tokenize-rt/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "tokenize-rt" %}
+{% set version = "2.0.1" %}
+{% set hash = "3cfa481f51ec1c6a1791964b14e4589c38d8877f2c74e5e50fe3e79dcf5f5bfa" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install
```

Comment:
can you switch this with `python -m pip install --no-deps --ignore-installed .`

Reply:
No problem.